ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -267,8 +267,6 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       BASELINE_CAPABILITY_SET: None
-      UPGRADE_CHECK_RUN_TAGS: not @console and not @storage
```

Comment:
Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml#L139 ?